recipes/libspnav/build.sh

```diff
@@ -0,0 +1,3 @@
+./configure --prefix="$PREFIX" --libdir="$PREFIX/lib"
+make CC="${CC}" AR="${AR}"
```

Comment:
I rechecked if we need anything else and saw https://github.com/FreeSpacenav/libspnav/blob/libspnav-0.2.3/Makefile.in#L8-L9. Could you (just for good measure, i.e., to avoid pulling in unwanted system dependencies) make this
```sh
make CC="${CC}" AR="${AR}" incpaths= libpaths=
```
?